CONTRIBUTING.md

```diff
@@ -242,3 +242,50 @@ INGRESS_HOST="$(kubectl -n istio-system get service istio-ingressgateway -o json
 echo "$INGRESS_HOST"
 curl -v "http://$INGRESS_HOST"
 ```
+
+### (Optional) Building and Running Individual Services Locally
+
+Each service runs in its own docker container; you can view the container images pushed to your GCP project's [Container Registry](https://pantheon.corp.google.com/gcr/images/). Instead of using something like GCP Cloud Build, you can also build and run each container locally.
```

Comment:
please change pantheon internal links to console.cloud.google.com counterparts (available externally).
We should not reference internal links on github

Reply:
Ah, my bad. Replaced.